packages/bits/src/lib/date-picker/date-picker-inner.component.ts

```diff
@@ -75,6 +75,8 @@ export class DatePickerInnerComponent implements AfterContentInit, OnInit, OnCha
 
     private modes: string[] = ["day", "month", "year"];
 
+    get role(): string { return this.inline ? "application" : "dialog"; }
```

Comment:
I'm not sure that `application` applies here. The MDN web docs say it should only be used to define very dynamic, desktop-like web applications.

![image](https://user-images.githubusercontent.com/10159693/113361433-9d6fa980-9311-11eb-871b-f49504f10957.png)
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Application_Role

Reply:
I used this resource: https://www.w3.org/TR/wai-aria-1.1/#application as a single source of truth

Reply:
okay